Dockerfile-clion

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:cosmic
```

Comment:
We should have a comment at the top with a reference to what this dockerfile does, that it is taken from the stackoverflow article.

Reply:
The same for the other dockerfile and the docker-compose file